tests/pulseaudio_dbus_spec.lua

```diff
@@ -201,4 +207,30 @@ describe("PulseAudio with DBus", function ()
                     end
                   end
            end)
+
+           it("Will set the next port as the ActivePort", function()
```

Comment:
I think that we should test using both `get_active_port` and `set_active_port` here. The two functions have no coverage at the moment.